ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.15.yaml

```diff
@@ -0,0 +1,92 @@
+base_images:
+  ansible:
+    name: "4.15"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.15"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.15"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.15"
+  tools:
+    name: "4.15"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.15"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      relative: 1
+      version_bounds:
+        lower: 4.15.0-0
+        upper: 4.16.0-0
+  target:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.15.0-0
+        upper: 4.16.0-0
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
```

Comment:
@jianli could u help add two more profiles(connected+disconnected) for tp cluster? we plan to support it in z-stream upgrade ci from 4.15. 

cc @shellyyang1989 @jianlinliu to help identify which tp profile could be a candidate for the upgrade path.

Reply:
Can I use `aws-ipi-disconnected-private-tp-f28`? and could you please help find out a connected profile?

Reply:
How about this two:
```
- as: azure-ipi-proxy-tp-arm-f28
- as: aws-ipi-disconnected-private-tp-arm-f28 
```

Reply:
lgtm, thanks

Reply:
ok, then I will add one for amd and the other one for arm, do you think that good for us?

Reply:
@JianLi-RH Looks like we don't have tp profile on arm now. So adding them on amd should be fine.

Reply:
Done with below 2 cases:
1. aws-ipi-disconnected-private-tp-f28
2. azure-ipi-proxy-tp-f28

Reply:
i added comments for test chain just now.
pls help trigger the test job against the two TP profiles after update since it's the first time to run upgrade ci on TP cluster, thanks.